py_zipkin/transport/firehose_handler.py

```diff
@@ -0,0 +1,70 @@
+# -*- coding: utf-8 -*-
+import socket
+
+import clog
+import staticconf
+
+
+ZIPKIN_CONFIG_NAMESPACE = 'zipkin'
+ZIPKIN_CONFIG_FILE = '/nail/srv/configs/zipkin.yaml'
+ZIPKIN_CONFIG_MIN_RELOAD_INTERVAL = 5
+
+ZIPKIN_FIREHOSE_IP = '169.254.255.254'
+ZIPKIN_FIREHOSE_PORT = 20182
+
+ZIPKIN_DEFAULT_FIREHOSE_ENABLED = False
+
+
+class FirehoseHandler(object):
+    """A configurable transport handler for firehose mode (CEP935)
+
+    This handler implements truth value testing so that it can be enabled,
+    disabled, or removed, while keeping the py_zipkin logic simple.
+
+    When called with a thrift-encoded zipkin message (typically a
+    thrift-encoded list of spans), it will forward this message over UDP to a
+    proxy, which generally runs as a daemon on the local host.
+    """
+
+    def __init__(self):
+        self.zipkin_namespace = staticconf.NamespaceReaders(
```

Comment:
I'd prefer if this did not require staticconf. I don't want to dep PyStaticConfiguration in py_zipkin.

Reply:
I feel like staticconf is an ok dependency. At least it's opensourced on github and it's maintaned (yelp-clog is not).

Also it's not yelp specific and should work for anyone.

Reply:
It's not yelp-specific, but it is specific to our use-case of disabling and enabling the transport handler, which is definitely not the normal case. If this is brought into a separate repo and/or as an extras_require, then i'm fine with it. Otherwise, I don't want to suddenly make people pull in a useless dependency.